config/crowdscores.yml

```diff
@@ -7,6 +7,8 @@ backend-developers:
     - jcalero
     - johtso
     - siminio
+    - glebanatalia
+    - 115100
```

Comment:
Given that this is supposed to be a string, I'm not sure it will work without quoting it.